### DISCUSSION TITLE
KIN KRE calculation tool

### DISCUSSION THREAD

**Discussion by Polype01** [kinfoundation/kin-rfcs#164] [KRE 4.0 Discussion]
Will the KIN Foundation provide an online tool to allow developers to simulate the amount of their KIN KRE payouts?

**Comment by hann228** [comment]
Yes a calculator is currently under development.  It may provide just an estimate rather than a detailed simulation for ease of use.

**Reply by Polype01** [reply]
If the estimate is pretty accurate and if it includes many selection parameters criterias, then it is good. (Ex: app with < 500 users, app with > 500 users….)
